.github/pull_request_template.md

```diff
@@ -0,0 +1,87 @@
+# Pull Request Title
+## Fixes \#(your issue number)
+
+<br>
+
+# Change Description
+<em>
+Please write a summary of the change and specify which Issue this fixes. This should cover the motivation for making the change and provide almost enough context to review the change in isolation. While the issue description is a good place to discuss the failure/requirements, this is the ideal space for explaining why this design fixes the issue and why you designed it the way you did.
+
+<br>
+
+- If there are important design decisions, particularly those that leverage functionality outside this change or were caused by existing limitations, they should be noted here.
+- If there is a design specification which relates to this PR, please provide a link to it here.
+- If there are any dependencies (third-party or internal downstream) which are added or impacted, please note them here. And if their limitations informed your design, please note how they did so here (particularly if this information was not captured in a design specification).
+</em>
+
+Text goes here
+
+
+
+
+<br>
+
+# Test Description
+<em>
+Please describe the tests you have written that verify the enhancement works or that the bug fix has been remedied. For a bug fix, please ensure a complete description of the failure is available here or in the issue such that a reviewer can verify that the testing is suitable. This is a good place to highlight any manual testing you may have had to perform. 
+
+Please describe any operating system limitations placed on the testing (if appropriate).
+
+If your tests fix bugs, please explain that in the comment you provided in the test file and please record the mapping here e.g.
```

Comment:
Not limited to bugs this is also to describe closing of features if defined within issues.

Reply:
Agree, expanding text